src/Service/ReleaseNotes.php

```diff
@@ -27,28 +27,27 @@
 
 class ReleaseNotes
 {
-    protected $releaseNotesDir = __DIR__ . '/../../Data/ReleaseNotes/';
+    protected string $releaseNotesDir = __DIR__ . '/../../Data/ReleaseNotes/';
 
     /**
-     * @return array
+     * @return array<string, string[]>
      */
     public function getAllReleaseNoteNames(): array
     {
-        /** @var \Symfony\Component\HttpFoundation\File\File[] $directories */
-        $directories = Finder::create()->directories()->in($this->releaseNotesDir)->sortByName();
+        /** @var \Symfony\Component\HttpFoundation\File\File[] $finder */
+        $finder = Finder::create()->directories()->in($this->releaseNotesDir)->sortByName();
         $result = [];
-        foreach ($directories as $directory) {
-            $dir = $directory->getRealPath() . DIRECTORY_SEPARATOR;
+        foreach ($finder as $singleFinder) {
```

Comment:
This is definitely an automated rename gone awry.  The original names were much clearer.

Reply:
Agree